packages/admin-ui/src/combobox/combobox.state.ts

```diff
@@ -39,7 +39,7 @@ export function useComboboxState<T>(
 
   // filters matches when input changes
   useEffect(() => {
-    if (!list) return
+    if (!list || list.length === 0) return
```

Comment:
I would instantiate the list prop like this: `list = []` and just check if the length is equal to 0. Just a suggestion, feel free to accept
```suggestion
    if (list.length === 0) return
```